ui/src/helpers/player-helper.js

```diff
@@ -3,6 +3,13 @@ const PlayerHelper = function() {
   const _self = this;
   const urlParams = new URLSearchParams(window.location.search);
   const isPreview = urlParams.get('preview') === '1';
+  const staticTemplateIds = [
```

Comment:
This sort of hard coding can't be used - lets discuss the problem and see how we can come up with an alternative solution.